tests/test_generate_success.py

```diff
@@ -0,0 +1,61 @@
+# Global imports
+"""Test the generate_config function in a perfect situation."""
+import pytest
+from gatorgrade.generate.generate import generate_config
+
+
+@pytest.fixture(name="testing_dir")
+def setup_files(tmp_path):
+    """Setup directory for tests"""
+    # Given this file structure
+    src_dir = tmp_path / "src"
+    src_dir.mkdir()
+
+    python_file = src_dir / "test.py"
+    python_file.write_text("import sys")
+
+    writing_dir = tmp_path / "writing"
+    writing_dir.mkdir()
+
+    reflection = writing_dir / "reflection.md"
+    reflection.write_text("# Reflection")
+    yield tmp_path
+
+
+def test_generate_config_create_gatorgrade_yml(testing_dir):
+    """Test to see that gatorgrade_yml file exist in file structure"""
+    # When generate_config is called
+    generate_config(["src"], testing_dir)
+    # Then gatorgrade.yml is created
+    gatorgrade_yml = testing_dir / "gatorgrade.yml"
+    assert gatorgrade_yml.is_file()
+
+
+def test_generate_config_creates_gatorgrade_yml_with_dir_in_user_input(testing_dir):
+    """Test to see if input matches directory"""
+    # When generate_config is called
+    generate_config(["src"], testing_dir)
+    # Then gatorgrade.yml is created
+    gatorgrade_yml = testing_dir / "gatorgrade.yml"
+    assert "src/test.py" in gatorgrade_yml.open().read()
+
+
+def test_generate_config_creates_gatorgrade_yml_without_dir_not_in_user_input(
+    testing_dir,
+):
+    """Test to see if input does not match directory"""
+    # When generate_config is called
+    generate_config(["writing"], testing_dir)
+    # Then gatorgrade.yml is created
+    gatorgrade_yml = testing_dir / "gatorgrade.yml"
+    assert "src/test.py" not in gatorgrade_yml.open().read()
```

Comment:
Hey Anthony, thanks for your hard work. I was having exactly the same issue yesterday. My tests were passing on LinuX and macOS but not on Windows because Windows uses different path. To resolve this issue, you might try using the `os.path.normpath("src/test.py")` instead of `"src/test.py"`.

Reply:
Also, don't forget to update your branch to the latest main branch to get the updated `generate.py` file.

Reply:
thank you!